tests/testthat/test-print.R

```diff
@@ -6,3 +6,7 @@ test_that("print works", {
       generate(reps = 10, type = "permute")
   ))
 })
+
+test_that("print method fits linewidth with many predictors (#543)", {
+  expect_snapshot(specify(mtcars, mpg ~ cyl + disp + hp + drat + wt + qsec))
+})
```

Comment:
Note that `test_that()` runs inside of `local_reproducible_output()`, so this test should always have a console width of 80.